gems/pending/util/system/evm_watchdog.rb

```diff
@@ -35,6 +39,13 @@ def self.check_evm
     end
   end
 
+  def self.check_for_update
+    return unless File.exist?(UPDATE_FILE)
+    packages = File.read(UPDATE_FILE).split(",")
```

Comment:
It's probably really unlikely but could this read while the other is trying to write?  Is a flock overkill? http://ruby-doc.org/core-2.2.5/File.html#method-i-flock